sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -133,6 +144,7 @@ def _create_appconfig_pipeline(self, credential, base_url=None, aad_mode=False,
                 DistributedTracingPolicy(**kwargs),
                 HttpLoggingPolicy(**kwargs),
                 ContentDecodePolicy(**kwargs),
+                RequestIdPolicy(**kwargs),
```

Comment:
I don't think `RequestIdPolicy` should be this far down the pipeline - pretty sure this should be before the retry policy - can you check the default generated pipeline to confirm?

Reply:
It is on top of the stack in the default pipeline, I'll change the sequence here.